scripts/standard_configure.sh

```diff
@@ -32,8 +32,15 @@ export PYTHON_EXECUTABLE=${PYTHON_EXECUTABLE:-"/usr/bin/python3"}
 # Set applications to compile
 export KRATOS_APPLICATIONS=
 add_app ${KRATOS_APP_DIR}/LinearSolversApplication
-add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
 add_app ${KRATOS_APP_DIR}/FluidDynamicsApplication
+add_app ${KRATOS_APP_DIR}/ConvectionDiffusionApplication
+add_app ${KRATOS_APP_DIR}/CoSimulationApplication
+add_app ${KRATOS_APP_DIR}/MappingApplication
+add_app ${KRATOS_APP_DIR}/FSIApplication
+add_app ${KRATOS_APP_DIR}/RomApplication
+add_app ${KRATOS_APP_DIR}/StructuralMechanicsApplication
+add_app ${KRATOS_APP_DIR}/ParticleMechanicsApplication
+
 
```

Comment:
It's easier to have your own `custom_configure.sh` so that these changes don't sneak into git.

Reply:
yes, modifying this file is a reason for blocking the PR